### PR TITLE
[IA-1292] Remove redundant files in docker folder

### PR DESCRIPTION
**Pull request by kyuksel**
The files that have been removed/kept are listed on this [wiki page](https://broadworkbench.atlassian.net/wiki/spaces/IA/pages/195264532/2020-01-22+Remove+unused+Leo+docker+files).

### CODE REVIEW

**Comment by Qi77Qi** [comment]
I think there're more files can be removed..but these files look safe to remove

**Comment by kyuksel** [comment]
> I think there're more files can be removed..but these files look safe to remove

Certainly more to remove. I just wanted to test as I remove them.

**Comment by kyuksel** [comment]
`ImportError: cannot import name UnrewindableBodyError`

jenkins retest

**Comment by kyuksel** [comment]
`fiab-start` failed. Looks unrelated.

jenkins retest

File: docker/build.sh [hunk 1]
```diff
@@ -184,11 +184,6 @@ function docker_cmd()
...
         fi
 
-        # builds the juptyer notebooks and rstudio docker images that go on dataproc clusters
-        # Stop building jupyter and rstudio images in leonardo repo as we move the process to terra-docker repo
-#        bash ./docker/build-tool.sh -i jupyter -r "${NOTEBOOK_REPO}" -t "${DOCKER_TAG}"
-#        bash ./docker/build-tool.sh -i rstudio -r "${NOTEBOOK_REPO}" -t "${DOCKER_TAG}"
-
         # Build the UI if specified.
         if $BUILD_UI; then
           bash ./ui/build.sh build "${NOTEBOOK_REPO}" "${DOCKER_TAG}"
```
Comment (Qi77Qi) [inline docker/build.sh:189]: would be nice to update this script to only publish leonardo images to gce only as well at some point

**Review by Qi77Qi — APPROVED** [review]
👍  once ci passes

File: docker/build.sh (hunk 1 above)
Comment (rtitle) [reply docker/build.sh:189]: Yeah.. I feel like `build.sh` could be simplified a lot, maybe best done as a separate PR

**Review by rtitle — APPROVED** [review]
I think this all looks good

**Pull request merged**